### ISSUE TITLE
Truncation of error message

### ISSUE THREAD

**Issue by mgoelzer** [filecoin-project/dealbot#139]
I see a lot of failed deals where the `/tasks` output is truncating the error to just `retrieval deal failed: `.  But when I ran a couple of these manually on the Lotus CLI, I got longer error messages on the console, e.g., `ERROR: retrieval failed: Retrieve: Retrieval Error: Unexpected deal response status: DealStatusErrored`.  The request in this issue is whether we can fix the truncation so that the full error message is shown.

Here's an example of a `/tasks` array element with a truncated "ErrorMessage" field:

```
{
  "UUID": "0205730c-2536-40f1-9afb-3c48f6cfb255",
  "Status": 4,
  "WorkedBy": "38d9a260",
  "Stage": "FirstByteReceived",
  "CurrentStageDetails": {
    "Description": "First byte of data received from miner",
    "ExpectedDuration": "a few seconds, or several hours when unsealing",
    "Logs": []
  },
  "PastStageDetails": [
    {
      "Description": "Miner is online",
      "ExpectedDuration": "a few seconds",
      "Logs": [
        {
          "Log": "Tipset successfully fetched",
          "UpdatedAt": 1621884538179313700
        },
        {
          "Log": "Miner Info successfully fetched",
          "UpdatedAt": 1621884538186197500
        },
        {
          "Log": "Miner address validated",
          "UpdatedAt": 1621884538190939000
        },
        {
          "Log": "Connected to miner",
          "UpdatedAt": 1621884538195835000
        }
      ],
      "UpdatedAt": 1621884538195835000
    },
    {
      "Description": "Miner responds to query ask",
      "ExpectedDuration": "a few seconds",
      "Logs": [
        {
          "Log": "Miner Offer Received",
          "UpdatedAt": 1621884538246455300
        }
      ],
      "UpdatedAt": 1621884538246455300
    },
    {
      "Description": "Send retrieval to miner",
      "ExpectedDuration": "",
      "Logs": [
        {
          "Log": "deal sent to miner",
          "UpdatedAt": 1621884538258789400
        }
      ],
      "UpdatedAt": 1621884538258789400
    },
    {
      "Description": "Miner accepts deal",
      "ExpectedDuration": "a few seconds",
      "Logs": []
    }
  ],
  "StartedAt": 1621884537396189700,
  "RunCount": 1,
  "ErrorMessage": "retrieval deal failed: ",
  "RetrievalTask": {
    "Miner": "f023985",
    "PayloadCID": "bafykbzacedkj4cwdigfic3rmr3kxjhcoq25v6loch45pxzu7wdxr2zhud2xkc",
    "CARExport": false
  }
}
```

Here's an example of Lotus CLI console output with a full error message:

```
$ lotus client retrieve --miner f023870 bafykbzacecqe643ksdept3qnkzh4hrtgwbvl5insg2paqy7zskuroduvt6l3k /dev/null
2457938 /mnt/hdd/ubuntu-home/lotus-daemon.log
> Recv: 0 B, Paid 0 FIL, ClientEventOpen (DealStatusNew)
> Recv: 0 B, Paid 0 FIL, ClientEventDealProposed (DealStatusWaitForAcceptance)
> Recv: 1.049 KiB, Paid 0 FIL, ClientEventBlocksReceived (DealStatusWaitForAcceptance)
> Recv: 53.06 KiB, Paid 0 FIL, ClientEventBlocksReceived (DealStatusWaitForAcceptance)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventBlocksReceived (DealStatusWaitForAcceptance)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventDealAccepted (DealStatusAccepted)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventPaymentChannelAddingFunds (DealStatusPaymentChannelAddingInitialFunds)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventPaymentChannelReady (DealStatusPaymentChannelAllocatingLane)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventLaneAllocated (DealStatusOngoing)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventPaymentRequested (DealStatusFundsNeeded)
> Recv: 1.052 MiB, Paid 0 FIL, ClientEventSendFunds (DealStatusSendFunds)
> Recv: 1.052 MiB, Paid 0.000000000002097152 FIL, ClientEventPaymentSent (DealStatusOngoing)
> Recv: 1.052 MiB, Paid 0.000000000002097152 FIL, ClientEventPaymentRequested (DealStatusFundsNeeded)
> Recv: 1.052 MiB, Paid 0.000000000002097152 FIL, ClientEventUnknownResponseReceived (DealStatusFailing)
> Recv: 1.052 MiB, Paid 0.000000000002097152 FIL, ClientEventCancelComplete (DealStatusErrored)
> Recv: 0 B, Paid 0 FIL, ClientEventOpen (DealStatusNew)
ERROR: retrieval failed: Retrieve: Retrieval Error: Unexpected deal response status: DealStatusErrored
```

**Comment by willscott** [comment]
fixed by #140 hopefully?

**Comment by mgoelzer** [comment]
Looks like it will be, yes!
